packages/phoenix-event-display/src/tests/managers/url-options-manager.test.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @jest-environment jsdom
+ */
+import { EventDisplay } from '../../event-display';
+import { URLOptionsManager } from '../../managers/url-options-manager';
+import { Configuration } from '../../lib/types/configuration';
+
+jest.mock('../../event-display');
+
+describe('URLOptionsManager', () => {
+  let urlOptionsManager: URLOptionsManager;
+  let eventDisplay: EventDisplay;
+  let configuration: Configuration;
+
+  beforeEach(() => {
+    eventDisplay = new EventDisplay();
+    urlOptionsManager = new URLOptionsManager(eventDisplay, configuration);
+  });
+
+  it('should create an instance', () => {
+    expect(urlOptionsManager).toBeTruthy();
+  });
+
+  it('should get options from URL set through query parameters', () => {
+    expect(urlOptionsManager.getURLOptions()).toBeInstanceOf(URLSearchParams);
+  });
+});
```

Comment:
There are other main public methods that seem to be not tested?

Reply:
Sure, done. ✅

Just this one remains due to this line being executed -
https://github.com/HSF/phoenix/blob/50d629e7dbf4432e3f1dd8b9a49466b53ecc0fc8/packages/phoenix-event-display/src/managers/url-options-manager.ts#L62

Reply:
This is the main issue: `Cannot read properties of undefined (reading 'addLoadableItem')`

it's just that this method is not callable

https://github.com/HSF/phoenix/blob/50d629e7dbf4432e3f1dd8b9a49466b53ecc0fc8/packages/phoenix-event-display/src/managers/url-options-manager.ts#L79

Reply:
That's because you are mocking the event display import I think. You can either mock these functions of the event display or use the event display as is (if that works).

Reply:
I can't use EventDisplay as it is without mocking due to this dependency hierarchy: `EventDisplay <- UIManager <- ThreeManager <- WebGLRenderer`. By default, `eventDisplay` is undefined due to this.